Archives/VolFile.cpp

```diff
@@ -6,33 +6,29 @@
 #include <algorithm>
 #include <climits>
 
-
 namespace Archives
 {
 	VolFile::VolFile(const char *fileName) : ArchiveFile(fileName)
 	{
-		if (MemoryMapFile(fileName)) {
-			throw std::runtime_error("Could not open vol file " + std::string(fileName) + ".");
-		}
+		archiveFileReader = std::make_unique<FileStreamReader>(fileName);
```

Comment:
I was just learning about constructor initializer lists when making the FileSliceReader, so good timing about posting this. I'll apply this suggestion in VolFile, ArchiveUnpacker, and ClmFile in the current open branch. The only reason they were wrapped in the unique pointer is because I couldn't figure out how to initialize them properly in the constructor earlier.